sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -610,8 +610,9 @@ struct _pi_context : _pi_object {
   std::unordered_map<ze_device_handle_t, std::list<ze_command_list_handle_t>>
       ZeCopyCommandListCache;
 
-  // Retrieves a command list for executing on this device along with
-  // a fence to be used in tracking the execution of this command list.
+  // Retrieves a command list for executing on this device.
+  // Depending on setting, this could be a standard or immediate commandlist.
+  // When using standard commandlists a fence is also retrieved.
```

Comment:
The "fence" is just a part of the "pi_command_list_ptr_t" so this comment should be removed. Please mark in the definition of the pi_command_list_ptr_t that "fence==nullptr" is what identifies an imm command-list

Reply:
OK